homeassistant/components/duotecno/const.py

```diff
@@ -1,3 +1,19 @@
 """Constants for the duotecno integration."""
+from typing import Final
 
-DOMAIN = "duotecno"
+from homeassistant.components.climate import HVACMode
+
+DOMAIN: Final = "duotecno"
+
+HVACMODE: Final = {
+    0: HVACMode.OFF,
+    1: HVACMode.HEAT,
+    2: HVACMode.COOL,
+}
+
+PRESETMODES: Final = {
+    "Sun": 0,
+    "Half Sun": 1,
+    "Moon": 2,
+    "Half Moon": 3,
```

Comment:
Made the modes snakecase and add translations like

https://github.com/home-assistant/core/blob/2a443648fc5da6dbdcce5713c21ad63b70ad989d/homeassistant/components/tado/strings.json#L50